src/Interactive/EditorFeatures/Core/Extensibility/Interactive/ResetInteractive.cs

```diff
@@ -104,11 +104,12 @@ internal Task Execute(IInteractiveWindow interactiveWindow, string title)
                 await interactiveEvaluator.SetPathsAsync(referenceSearchPaths, sourceSearchPaths, projectDirectory).ConfigureAwait(true);
             }
 
-            await interactiveWindow.SubmitAsync(new[]
+            var importReferencesCommand = referencePaths.Select(_createReference);
```

Comment:
There is currently not. It'd certainly be useful to have it. We should do that in U3. Can you file an issue?